server/core/region.go

```diff
@@ -304,6 +304,9 @@ func (r *RegionInfo) GetID() uint64 {
 
 // GetMeta returns the meta information of the region.
 func (r *RegionInfo) GetMeta() *metapb.Region {
+	if r == nil {
+		return nil
+	}
```

Comment:
why need this?

Reply:
If not will panic in this [line](https://github.com/pingcap/pd/pull/2022/files#diff-02ca8f1595ce93b75591586acd227047R244).